src/components/EventCard/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Box, Image, Heading, Button, Grid, ResponsiveContext } from 'grommet';
+import { Link as GatsbyLink } from 'gatsby';
+
+const NavLink = styled(GatsbyLink)`
+  text-decoration: none;
+`;
+
+export const EventCard = ({ timeframe, link, image, title }) => (
+  <Box elevation="medium" pad="large" direction="row" wrap gap="large">
+    <NavLink to={link}>
+      <Box direction="row-responsive" gap="large" align="center">
+        <Box justify="between" direction="row-responsive">
+          <Box gap="medium">
+            <Box gap="small">
+              <Heading color="text" margin="none">
+                {title}
+              </Heading>
+              <Heading color="text" margin="none" level="3">
+                {timeframe}
+              </Heading>
+            </Box>
+            <Box direction="row">
+              <Button primary label="View Events Page"></Button>
+            </Box>
+          </Box>
+          <Box>{image && <Image fit="contain" src={image} />}</Box>
+        </Box>
+      </Box>
+    </NavLink>
+  </Box>
+);
+EventCard.propTypes = {
+  timeframe: PropTypes.string,
+  link: PropTypes.string,
+  image: PropTypes.string,
+  title: PropTypes.string,
+};
+
+const columns = {
+  small: ['auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto', 'auto'],
+  xlarge: ['auto', 'auto'],
+};
+
+const rows = {
+  small: ['auto', 'auto', 'auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto'],
+  xlarge: ['auto'],
+};
+
+export const PastEventCard = ({
+  children,
+  overrideColumns,
```

Comment:
similar comments to your blog review about override* and how you're handling the columns/rows props